src/components/Calculator.js

```diff
@@ -1,125 +1,123 @@
-              onClick={this.handleClick}
-            >
-              &divide;
-            </button>
-            <button type="button" className="button" onClick={this.handleClick}>
-              7
-            </button>
-            <button type="button" className="button" onClick={this.handleClick}>
-              8
-            </button>
-            <button type="button" className="button" onClick={this.handleClick}>
-              9
-            </button>
-            <button
-              type="button"
-              className="button colored"
-              onClick={this.handleClick}
-            >
-              &times;
-            </button>
-            <button type="button" className="button" onClick={this.handleClick}>
-              4
-            </button>
-            <button type="button" className="button" onClick={this.handleClick}>
-              5
-            </button>
-            <button type="button" className="button" onClick={this.handleClick}>
-              6
-            </button>
-            <button
-              type="button"
-              className="button colored"
-              onClick={this.handleClick}
-            >
-              &minus;
-            </button>
-            <button type="button" className="button" onClick={this.handleClick}>
-              1
-            </button>
-            <button type="button" className="button" onClick={this.handleClick}>
-              2
-            </button>
-            <button type="button" className="button" onClick={this.handleClick}>
-              3
-            </button>
-            <button
-              type="button"
-              className="button colored"
-              onClick={this.handleClick}
-            >
-              +
-            </button>
-            <button
-              type="button"
-              className="button button_3"
-              onClick={this.handleClick}
-            >
-              0
-            </button>
-            <button type="button" className="button" onClick={this.handleClick}>
-              .
-            </button>
-            <button
-              type="button"
-              className="button colored"
-              onClick={this.handleClick}
-            >
-              =
-            </button>
-          </div>
+  return (
+    <div className="calc-container">
+      <div className="calc-wrap">
+        <div className="button-wrap">
+          <input className="button span_5 input" placeholder="0" />
+          <button type="button" className="button" onClick={handleClick}>
+            AC
+          </button>
+          <button type="button" className="button" onClick={handleClick}>
+            +/-
+          </button>
+          <button type="button" className="button" onClick={handleClick}>
+            %
+          </button>
+          <button
+            type="button"
+            className="button colored"
+            onClick={handleClick}
+          >
+            /
```

Comment:
- [ ] After the refactor your app is still required to work as expected. I am unable to perform **division** operations because of the sign used. The `operate.js` file has the **&#247;**, whereas you have this `/` sign, hence the error. Kindly fix this.

  If you do not have the proper division sign on your keyboard, you can use this HTML entity code: **`&#247;`**

  ![ayoub-divide](https://user-images.githubusercontent.com/36542986/158777412-4a135aad-df08-4646-8a2d-d315a387fd62.jpg)

Reply:
Fixed, Thank you